src/riak_kv_qry_compiler.erl

```diff
@@ -2229,6 +2247,50 @@ two_element_key_range_cannot_match_test() ->
         compile(DDL, Q, 100)
     ).
 
+group_by_one_field_test() ->
```

Comment:
I can only see one, that is covered here https://github.com/basho/riak_ql/pull/132/files#diff-53ba1ef30ed23e05f486e07b0a995b46R223